packages/amplication-plugin-api/src/pluginVersion/npm-plugin-version.service.ts

```diff
@@ -98,7 +92,7 @@ export class NpmPluginVersionService {
 
       return pluginsVersions;
     } catch (error) {
-      // TODO add error handling
+      console.error;
```

Comment:
It supposed to be `console.error(error);`
Btw, we have logger, so `logger.error(error);` wuold be even better.
Check billing.service.ts for example.

Reply:
I'll tackle the logger as a new PR once we merge this PR (that I'm working on during my off time) #5158